### PR TITLE
jinja2-git: init at 1.1.0

### PR DESCRIPTION
**Pull request by cpcloud**
###### Motivation for this change

This package adds jinja2_git to nixpkgs

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -3730,6 +3730,8 @@ in {
 
   jinja2 = callPackage ../development/python-modules/jinja2 { };
 
+  jinja2_git = callPackage ../development/python-modules/jinja2_git { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-packages.nix:3733]:
```suggestion
  jinja2-git = callPackage ../development/python-modules/jinja2-git { };
```

File: pkgs/development/python-modules/jinja2_git/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  version = "1.1.0";
+  pname = "jinja2-git";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "wemake-services";
+    repo = "jinja2-git";
+    # this is master, we can't patch because of poetry.lock :(
+    # luckily, there appear to have been zero API changes since then, only
+    # dependency upgrades
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/jinja2_git/default.nix:18]: Then we need to set the version number accordingly.

File: pkgs/development/python-modules/jinja2_git/default.nix [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, jinja2
+, poetry-core
+}:
+
+buildPythonPackage rec {
+  version = "1.1.0";
+  pname = "jinja2-git";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/jinja2_git/default.nix:10]:
```suggestion
  pname = "jinja2-git";
  version = "1.1.0";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 134889` at 3ccb4da8 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.jinja2_git</li>
    <li>python39Packages.jinja2_git</li>
  </ul>
</details>

**Pull request merged**